configs/common/FUScheduler.py

```
@@ -24,13 +24,27 @@ def FpRD(id, p):
     ret = (1 << 6) | (id << 2) | (p)
     return ret
 
+def RMiscRD(id, p):
```

Ruff (0.15.14)

[error] Function argument `id` is shadowing a Python builtin

(A002)